rust-toolchain.toml

```diff
@@ -1,2 +1,3 @@
 [toolchain]
 channel = "nightly"
+targets = [ "x86_64-unknown-linux-musl" ]
```

Comment:
This won't negatively impact development on Linux (e.g.: `nix develop -c cargo run`), right?

Reply:
Seems to be working fine for me